tests/integration/test_materialized_mysql_database/materialize_with_ddl.py

```diff
@@ -1141,14 +1141,14 @@ def materialized_database_support_all_kinds_of_mysql_datatype(clickhouse_node, m
             `v19` datetime(6) DEFAULT CURRENT_TIMESTAMP(6),
             `v20` TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
             `v21` TIMESTAMP(6) DEFAULT CURRENT_TIMESTAMP(6),
-            /* todo support */
-            # `v22` YEAR,
-            # `v23` TIME,
-            # `v24` TIME(3),
+            `v22` YEAR,
+            `v23` TIME,
+            `v24` TIME(6),
             # `v25` GEOMETRY,
```

Comment:
So `GEOMETRY` is not supported, right?

Reply:
GEOMETRY is supported, we map GEOMETRY to String, but clickhouse have not a series of GEOMETRY function, users can't use the value of GEOMETRY type in fact . so, I'm not sure what to do with it. Perhaps I should add this type to documentation（Add special tips） and tests？

Reply:
i will fix it later

Reply:
done